src/playlist.js

```diff
@@ -261,12 +261,13 @@ export const duration = function(playlist, endSequence, expired) {
   * playlist in which case, the targetDuration of the playlist is used
   * to approximate the durations of the segments
   *
-  * @param {Object} playlist a media playlist object
-  * @param {number} startIndex
-  * @param {number} endIndex
+  * @param {Object} durationList list to iterate over for durations.
+  * @param {number} defaultDuration duration to use for elements before or after the durationList
+  * @param {number} startIndex partsAndSegments index to start
+  * @param {number} endIndex partsAndSegments index to end.
```

Comment:
Suggestion: since this changed to an options object, we should rename the params to reflect that
```suggestion
  * @param {Object} options.durationList list to iterate over for durations.
  * @param {number} options.defaultDuration duration to use for elements before or after the durationList
  * @param {number} options.startIndex partsAndSegments index to start
  * @param {number} options.endIndex partsAndSegments index to end.
```